plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -57,6 +58,10 @@ func NewPersistentVolumeLabel() *persistentVolumeLabel {
 	}
 }
 
+func (l *persistentVolumeLabel) SetCloudConfig(cloudConfig io.Reader) {
```

Comment:
Ok thanks. If it were required, I was going to suggest you implement `Validate`, but it doesn't sound like we need that.